status-handler.js

```diff
@@ -10,6 +12,11 @@ const defaultLogger = {
   error: diagnostics('warehouse.ai-status-api:status-handler:error')
 };
 
+const BUILD_STARTED = 'build_started';
+const MSG_BUILD_QUEUED = 'Builds Queued';
+const MSG_FETCHED_TARBALL = 'Fetched tarball';
+const WEBHOOKS_CONC = 5;
+const WEBHOOK_TIMEOUT = 2000;
```

Comment:
Not blocking but the concurrency and timeout should be able to be configured via config file

Reply:
Makes sense.